### PR TITLE
Horse

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds horse dildos for the traitor captain to (never) buy. Can also show up in surplus crates

## Why It's Good For The Game

they wouldn't, I just wanted to make this PR for memes

## Changelog
:cl:
add: Horse dildos
/:cl:

### CODE REVIEW

File: beestation.dme [hunk 1]
```diff
@@ -2944,4 +2945,5 @@
 #include "interface\menu.dm"
 #include "interface\stylesheet.dm"
 #include "interface\skin.dmf"
+
```
Comment (ondrej008) [inline beestation.dme]: What is this?

File: code/game/objects/items/dildos.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//cause there ain't no cock like horse cock
+/obj/item/dildo
+	name 				= "dildo"
+	desc 				= "Floppy!"
```
Comment (ondrej008) [inline code/game/objects/items/dildos.dm]: Convert the tabs after the variable names to spaces, or face the wrath of DM.

**Review by ondrej008 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/dildos.dm [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	icon 				= 'icons/dildo.dmi'
```
Comment (ondrej008) [inline code/game/objects/items/dildos.dm]: You have added the file as icons/obj/dildo.dmi
Change this to that.

**Comment by ghost** [comment]
i intentionally wrote this to fuck with qwerty, but am I really that determined to fix code I never actually expected to work?

File: beestation.dme (hunk 1 above)
Comment (ondrej008) [reply beestation.dme]: Resolved.

**Comment by piterskiy2** [comment]
what the acutal fuck.
cease

**Comment by ondrej008** [comment]
You should add the normal dildo too, I'd say put it in dorms, or have a chance for crew members to get it in their backpack, with female crew members having a greater chance. Could be an antag item, too.
Also, the horse dildo isn't big enough.

**Comment by Oshibkka** [comment]
why.

**Comment by qwertyquerty** [comment]
no

**Pull request closed without merging**